app/home/route.ts

```diff
@@ -6,28 +6,20 @@ import Session from 'ember-simple-auth/services/session';
 
 import Analytics from 'ember-osf-web/services/analytics';
 
-import Controller from './controller';
-
 export default class Home extends Route {
-    @service analytics!: Analytics;
-    @service session!: Session;
-
-    async beforeModel(this: Home, transition: Transition) {
-        await super.beforeModel(transition);
+  @service analytics!: Analytics;
+  @service session!: Session;
 
-        if (this.session.isAuthenticated) {
-            this.transitionTo('dashboard');
-        }
-    }
+  async beforeModel(this: Home, transition: Transition) {
```

Comment:
You shouldn't need to give `this` a type in class methods -- TS can figure it out.